packages/amplify-prompts/src/flags.ts

```diff
@@ -1,8 +1,7 @@
 /**
  * If this flag is set, debug messages will be printed.
  */
-export const isDebug = process.argv.includes('--debug');
-
+export const isDebug = process.argv.includes('--debug') || process.env.AMPLIFY_DEV_INTERNAL_DEBUG === 'true';
```

Comment:
I'd just call it AMPLIFY_ENABLE_DEBUG_OUTPUT. It's end of the day available to anybody.